RevenueCatUI/CustomerCenter/Views/RelevantPurchasesListView.swift

```diff
@@ -120,47 +120,33 @@ struct RelevantPurchasesListView: View {
     var content: some View {
         ScrollViewWithOSBackground {
             LazyVStack(spacing: 0) {
-                if !customerInfoViewModel.hasPurchases {
-                    CompatibilityContentUnavailableView(
-                        self.viewModel.screen.title,
-                        systemImage: "exclamationmark.triangle.fill",
-                        description: Text(self.viewModel.screen.subtitle ?? localization[.tryCheckRestore])
-                    )
-                    .padding()
-                    .fixedSize(horizontal: false, vertical: true)
-                    .background(
-                        RoundedRectangle(cornerRadius: 8)
-                            .fill(
-                                Color(colorScheme == .light
-                                      ? UIColor.systemBackground
-                                      : UIColor.secondarySystemBackground)
-                            )
-                            .padding(.horizontal)
-                            .padding(.top)
-                    )
-                    .padding(.bottom, 32)
+                if !customerInfoViewModel.activeSubscriptionPurchases.isEmpty {
```

Comment:
This is changing from changing subs + non-subs to only subs. What's the reason?

Reply:
The empty state is for subscriptions now, and since there are two sections we only put it if there are no subs

Reply:
I see, makes sense